### ISSUE TITLE
A bug in to_dict() for class `TemporalDataset`

### ISSUE THREAD

**Issue by shimalice** [rsagroup/rsatoolbox#419]
**time_descriptors should be assigned in the method `to_dict()`**

In the definition of instance method `to_dict()` of class `TemporalDataset`, 
`data_dict['time_descriptors'] = self.channel_descriptors`
should be replaced by
`data_dict['time_descriptors'] = self.time_descriptors`

**To Reproduce**
To check
```

channel_names = ['Oz', 'O1', 'O2', 'PO3', 'PO4', 'POz']  # channel names
stimulus = [0, 1, 2, 3, 4, 0, 1, 2, 3, 4] # stimulus idx, each stimulus was presented twice

sampling_rate = 30 # in Hz
t = numpy.arange(-.5, 1.5, 1/sampling_rate) # time vector

n_observations = len(stimulus)
n_channels = len(channel_names)
n_timepoints = len(t)

measurements = numpy.random.randn(n_observations, n_channels, n_timepoints)  # random data

data = rsatoolbox.data.TemporalDataset(
    measurements,
    channel_descriptors={'names': channel_names},
    obs_descriptors={'stimulus': stimulus},
    time_descriptors={'time': t}
    )
data.to_dict()
```

output:
```
{'measurements': array([[[ 7.66058159e-02,  6.22145196e-01,  3.02147022e-01, ...,
...
 'descriptors': {},
 'obs_descriptors': {'stimulus': [0, 1, 2, 3, 4, 0, 1, 2, 3, 4]},
 'channel_descriptors': {'names': ['Oz', 'O1', 'O2', 'PO3', 'PO4', 'POz']},
 'time_descriptors': {'names': ['Oz', 'O1', 'O2', 'PO3', 'PO4', 'POz']},
 'type': 'TemporalDataset'}

```
 
 **_'time_descriptors': {'names': ['Oz', 'O1', 'O2', 'PO3', 'PO4', 'POz']},_**

**Comment by JasperVanDenBosch** [comment]
hi @shimalice , good catch! Do you feel like opening a PR for this? If not no worries I'll write a fix later

**Comment by shimalice** [comment]
Ok, I would like to open a Pull Request. Pls wait a second.

**Comment by JasperVanDenBosch** [comment]
Resolved with #420
